groups_create.go

```diff
@@ -0,0 +1,25 @@
+package zoom
+
+// CreateGroupsPath -v2 path for add groups
+const CreateGroupsPath = "/groups"
+
+// CreateGroupOption - options for creating a group
+type CreateGroupOption struct {
+	Name string `json:"name"`
+}
+
+// CreateGroup calls POST /groups
+func CreateGroup(opts CreateGroupOption) error {
+	return defaultClient.CreateGroup(opts)
+}
+
+// CreateGroup calls POST /groups
+// https://marketplace.zoom.us/docs/api-reference/zoom-api/methods/#operation/groupCreate
+func (c *Client) CreateGroup(opts CreateGroupOption) error {
```

Comment:
Upon successful group creation, the API will return a group object containing the fields `id`, `name`, and `total_members`. This function should return both that struct and an an error - see `meeting_create.go` as an example.